src/ivoc/mymath.cpp

```diff
@@ -26,14 +26,10 @@ static double inside(void*) {
 int nrn_feround(int);
 // return last rounding mode and set to given mode if 1,2,3,4.
 // order is FE_DOWNWARD, FE_TONEAREST, FE_TOWARDZERO, FE_UPWARD
-#if defined(HAVE_FENV_H) && defined(HAVE_FESETROUND)
-#include <fenv.h>
+#include <cfenv>
```

Comment:
If we switch to using the C++ header then we should also use the `std::` prefixes, as the functions are only **required** to be present inside the `std::` namespace: https://en.cppreference.com/w/cpp/header#C_compatibility_headers